plaid/institutions.go

```diff
@@ -42,6 +43,19 @@ type InstitutionStatusBreakdown struct {
 	ErrorInstitution float64 `json:"error_institution"`
 }
 
+type InstitutionTransactionsUpdates struct {
```

Comment:
Given the API updates, this name should be generalized and so it can be re-used for the other statuses.

Reply:
Struct name changed to more generic `InstitutionStatusFields`.